.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseTimePeriodParser.cs

```diff
@@ -717,7 +717,7 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             pr1 = this.config.TimeParser.Parse(ers[0], referenceTime);
             pr2 = this.config.TimeParser.Parse(ers[1], referenceTime);
 
-            if (pr1.Value == null || pr2.Value == null)
+            if (pr1.Value == null || pr2.Value == null || pr1.Text == pr2.Text)
```

Comment:
Which situation falls into this new clause?

Reply:
in Hindi, there are expressions like "morning-morning" meaning "early-morning" that are otherwise parsed as "from morning to morning"

Reply:
the change does not seem to affect other languages.
Is it better that we revert it and set the corresponding case as not supported?

Reply:
I'm Ok with this, just add a comment in the code explaining what it addresses.

Reply:
I'm Ok with this, just add a comment in the code explaining what it addresses.

Reply:
Please add the comment in the next PR, ok?